### ISSUE TITLE
Warning message for pro version in admin dashboard

### ISSUE THREAD

**Issue by DavidCaum** [dudaster/ele-custom-skin#880]
Hi,
Since few days, I get this warning on the top of top of my plugin admin page (repeated 3 times):
Warning: Attempt to read property "new_version" on bool in /homepages/36/d832874816/htdocs/artinprovence/wp-content/plugins/ele-custom-skin-pro/update.php on line 72
There's no problem on frontend.
Any idea of the issue?

WordPress 6.1.1
Elementor 3.8.1
Elementor Pro 3.8.2
Ele Custom Skin 3.1.7
Ele Custom Skin 3.2.4

**Comment by dudaster** [comment]
I’ll take a look

**Comment by Designbase** [comment]
Yeah there are major issues, breaking all pages where it is used...!

**Comment by Designbase** [comment]
OK Found it. It is not compatible with PHP version 8.1.8. Went back to 7.4 and that solved the issue.
Reason that it is not showing on the front page may be a cache plugin?

**Comment by DavidCaum** [comment]
It's weird but today, the warning has disappear...
The only update made since yesterday is WP Rocket (cache plugin).
For information, it runs with PHP 8.0.26

**Comment by trinermedia** [comment]
We are currently experiencing this error as well. However, it is actually unclear when or by what exactly the error message is triggered. So far it was mostly when a plugin is installed for the first time or during the update of a plugin.
By the way: These are probably reports of the same problem: #858 #847
